content/applications/inventory_and_mrp/inventory/shipping_receiving/reservation_methods/before_scheduled_date.rst

```diff
@@ -0,0 +1,153 @@
+but the scheduled date, for this example workflow, is five days from the order date.
+
+Since reservation is not until two days before the scheduled delivery, it will not reserve the
+products until then.
+
+.. note::
+   If there is **not** sufficient quantity of stock for the product included in the |SO|, the
+   :guilabel:`📈 (forecast graph)` icon is red, instead of green.
+
+   Instead of revealing the reserved number of units for the order, the :guilabel:`Availability`
+   tooltip reads :guilabel:`Reserved`, and reveals the available number of units (e.g., `0 Units`).
+
+   Additionally, unless there is a set replenishment or a live receipt, it also reads :guilabel:`No
+   future availability`, in red text.
+
+.. image:: before_scheduled_date/before-scheduled-date-availability-tooltip.png
+   :align: center
+   :alt: Confirmed sales order with product availability tooltip selected.
+
+Click the :guilabel:`Delivery` smart button to see the delivery order form.
+
+On the delivery order form, the status in the :guilabel:`Product Availability` field is listed as
+`Available`, in yellow text, instead of green. This is because there is sufficient stock on-hand for
+this order, but no quantity has been reserved yet.
+
+Note the :guilabel:`Scheduled Date` field, above the :guilabel:`Product Availability` field,
+displays the date five days from the order creation date. This indicates that the products are not
+reserved until three days from today's date (two days before the scheduled delivery date).
+
+.. image:: before_scheduled_date/before-scheduled-date-delivery-order-form.png
+   :align: center
+   :alt: Delivery order form with product availability and reserved quantity.
+
+In the :guilabel:`Operations` tab on the :guilabel:`Product` line, the numbers in the
+:guilabel:`Demand` column and the :guilabel:`Quantity` column do not match (in this case, the
+:guilabel:`Demand` column lists `10.00`, while the :guilabel:`Quantity` column lists `0`).
+
+The :guilabel:`Quantity` column lists `0` because the products aren't reserved until two days
+*before* their delivery date. Odoo automatically reserves the products once the scheduled date
+arrives, at which point the :guilabel:`Demand` and :guilabel:`Quantity` columns will match.
+
+.. tip::
+   If the products in the |SO| should be reserved *sooner* than the scheduled reservation date, the
+   reservation can be manually overridden. To manually reserve the products sooner than scheduled,
+   click :guilabel:`Check Availability` at the top of the form.
+
+   This turns the `Available` status in the :guilabel:`Product Availability` field green, and
+   changes the number in the :guilabel:`Quantity` column to match the :guilabel:`Demand` column.
+
+   Once ready, click :guilabel:`Validate`.
+
+.. seealso::
+   - :doc:`Manual reservation <../reservation_methods/manually>`
+   - :doc:`At confirmation reservation <../reservation_methods/at_confirmation>`
```

Comment:
No need to back out of this directory and move right back into it 😄, if I suggested this in the past I apologize, the path can simply be:
```suggestion
   - :doc:`Manual reservation <manually>`
   - :doc:`At confirmation reservation <at_confirmation>`
```

Reply:
Suggestion implemented ✅